### PR TITLE
Move pieces

### PR DESCRIPTION
**Pull request by priceat**

### CODE REVIEW

File: test/controllers/pieces_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class PiecesControllerTest < ActionController::TestCase
+  # test "the truth" do
+  #   assert true
```
Comment (kenmazaika) [inline test/controllers/pieces_controller_test.rb]: Solid! :)

Comment (priceat) [inline test/controllers/pieces_controller_test.rb]: Ken,

I had a question about the assert_equals portion of the test.

Although it worked, some of the tests I have been reviewing return the
updated params of the object updated (in this case, the piece). Is there a
better way to test the actual put => update method other than
assert_response :success?

Further, is it a good idea to hard code the new values, i.e. put :update,
id: piece.id, :piece => {x_coord: 3, y_coord: 4}

Or should I test a different method?

Thanks!

Tate

**Comment by wengs** [comment]
Tate, could you pull the code from master and solve the conflicts?

File: test/controllers/pieces_controller_test.rb (hunk 1 above)
Comment (kenmazaika) [inline test/controllers/pieces_controller_test.rb]: Looks great,Tate.

If you're returning JSON to the user, sometimes you'll want to verify the contents of the response itself.

Inside the test:

```
@response.body
```

Will be the content of the resulting request (i.e. the html or json that comes back).  If you're building an API that is hit via AJAX & jQuery, verifying the body of the method is probably a smart move.

All the code you have in the test is solid.

**Comment by priceat** [comment]
Weng, can you merge this request?

**Pull request merged**